### PR TITLE
Set PROMPT in /etc/zsh/zshrc

### PR DESCRIPTION
**Pull request by hiroakit**
Merging this PR resolves the following issues:

- #1285
- #1624 

By setting PROMPT in /etc/zsh/zshrc, added no unintended PROMPT viewing 
when running interactive zsh from bash.

MEMO:
~Occurring a pgp error in my machine.~ => Resolved it.

### CODE REVIEW

**Comment by Biswa96** [comment]
> Occurring a pgp error in my machine.

Use `gpg --recv-keys` with the public key from validpgpkeys array.

**Comment by hiroakit** [comment]
@Biswa96

Thank you. I would try it.

File: zsh/PKGBUILD [hunk 1]
```diff
@@ -13,6 +13,7 @@ source=("https://www.zsh.org/pub/zsh-${pkgver}.tar.xz"{,.asc}
         'config.guess::https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD'
         'config.sub::https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD'
         'zprofile'
+        'zshrc'
```
Comment (lazka) [inline zsh/PKGBUILD]: bump pkgrel

Comment (hiroakit) [reply zsh/PKGBUILD]: @lazka 

Thank you for your reviewing.
I did it. Please confirm.

**Comment by lazka** [comment]
lgtm

**Comment by hiroakit** [comment]
Fixed my pgp error.
> Occurring a pgp error in my machine.

I have successfully installed the zsh package which created with this PR. 
And the PROMPT display is as intended.

**Comment by Alexpux** [comment]
@hiroakit @lazka this create conflict with grml-zsh-config package. I'm reverting commit

**Comment by lazka** [comment]
ok, sorry

**Comment by hiroakit** [comment]
> this create conflict with grml-zsh-config package. I'm reverting commit
Thank you for your reverting.
I didn't know about `grml-zsh-config` package. I'll try making a PR once more.

**Pull request merged**